docs/griptape-framework/drivers/src/local_ruleset_driver.py

```diff
@@ -0,0 +1,7 @@
+from griptape.drivers import LocalRulesetDriver
+from griptape.rules import Ruleset
+
+ruleset = Ruleset(
+    name="my_local_ruleset.json",
+    ruleset_driver=LocalRulesetDriver(persist_dir="/path/to/ruleset/dir", raise_not_found=False),
```

Comment:
Would like this to run as well. Can you add some set up file system logic before we load in?